### ISSUE TITLE
Error with FloatingMenuPluginKey between Tiptap and Ngx-Tiptap

### ISSUE THREAD

**Issue by GitHubish** [sibiraj-s/ngx-tiptap#11]
When I build my project I have an error with Ngx-tiptap and Tiptap. I am in version Ngx-Tiptap 3.0.2 and @tiptap/extension-floating-menu in version 2.0.0-beta.21

```
./node_modules/ngx-tiptap/fesm2015/ngx-tiptap.js:162:33-54 - Error: export 'FloatingMenuPluginKey' (imported as 'FloatingMenuPluginKey') was not found in '@tiptap/exten
sion-floating-menu' (possible exports: FloatingMenu, FloatingMenuPlugin, FloatingMenuView, default)

./node_modules/ngx-tiptap/fesm2015/ngx-tiptap.js:219:33-52 - Error: export 'BubbleMenuPluginKey' (imported as 'BubbleMenuPluginKey') was not found in '@tiptap/extension
-bubble-menu' (possible exports: BubbleMenu, BubbleMenuPlugin, BubbleMenuView, default)
```

Do you also encounter this problem?
Thanks

**Comment by sibiraj-s** [comment]
Tiptap removed those in the recent versions, Temporarily pin your dependencies to 

```
"@tiptap/core": "2.0.0-beta.93"
"@tiptap/extension-bubble-menu": "2.0.0-beta.25"
"@tiptap/extension-floating-menu": "2.0.0-beta.19"
```

I will release a patch by tomorrow.

**Comment by sibiraj-s** [comment]
Fixed in v3.0.3

**Comment by salmin89** [comment]
@sibiraj-s are you able to release a patch to v1.x as well?

We are currently on Angular 9 and while we will upgrade to the latest some day, I am currently trying to make a POC for tiptap at work.

The first issue I ran into was covered here: https://github.com/sibiraj-s/ngx-tiptap/issues/10
and you recommended us to downgrade to v1.x
